test/unit/search/search_collection_test.cpp

```diff
@@ -45,162 +51,153 @@ TYPED_TEST_SUITE(search_string_test, fm_index_string_types, );
+        EXPECT_EQ(uniquify(search("ACGG"_dna4, this->index, cfg)), hits_result_t{});
     }
 
     {
         // successful and unsuccesful exact search using short version of max_total_error
-        seqan3::configuration const cfg = seqan3::search_cfg::max_error{seqan3::search_cfg::total{0}};
-        EXPECT_EQ(seqan3::uniquify(search("ACGT"_dna4, this->index, cfg)), (hits_result_t{{0, 0}, {0, 4}, {0, 8},
-                                                                                          {1, 0}, {1, 4}, {1, 8}}));
-        EXPECT_EQ(seqan3::uniquify(search("ACGG"_dna4, this->index, cfg)), (hits_result_t{}));
+        seqan3::configuration const cfg =seqan3::search_cfg::max_error{seqan3::search_cfg::total{0}};
+        EXPECT_EQ(uniquify(search("ACGT"_dna4, this->index, cfg)), expected_hits);
+        EXPECT_EQ(uniquify(search("ACGG"_dna4, this->index, cfg)), hits_result_t{});
     }
 
     {
         // successful and unsuccesful exact search using max_total_error
-        seqan3::configuration const cfg = seqan3::search_cfg::max_error{seqan3::search_cfg::total{0},
-                                                                        seqan3::search_cfg::substitution{0},
-                                                                        seqan3::search_cfg::insertion{0},
-                                                                        seqan3::search_cfg::deletion{0}};
-        EXPECT_EQ(seqan3::uniquify(search("ACGT"_dna4, this->index, cfg)), (hits_result_t{{0, 0}, {0, 4}, {0, 8},
-                                                                                          {1, 0}, {1, 4}, {1, 8}}));
-        EXPECT_EQ(seqan3::uniquify(search("ACGG"_dna4, this->index, cfg)), (hits_result_t{}));
+        seqan3::configuration const cfg =seqan3::search_cfg::max_error{seqan3::search_cfg::total{0},
+                                                                       seqan3::search_cfg::substitution{0},
+                                                                       seqan3::search_cfg::insertion{0},
+                                                                       seqan3::search_cfg::deletion{0}};
+        EXPECT_EQ(uniquify(search("ACGT"_dna4, this->index, cfg)), expected_hits);
+        EXPECT_EQ(uniquify(search("ACGG"_dna4, this->index, cfg)), hits_result_t{});
     }
 
     {
         // successful and unsuccesful exact search using empty max_total_error_rate
-        seqan3::configuration const cfg = seqan3::search_cfg::max_error_rate{};
-        EXPECT_EQ(seqan3::uniquify(search("ACGT"_dna4, this->index, cfg)), (hits_result_t{{0, 0}, {0, 4}, {0, 8},
-                                                                                          {1, 0}, {1, 4}, {1, 8}}));
-        EXPECT_EQ(seqan3::uniquify(search("ACGG"_dna4, this->index, cfg)), (hits_result_t{}));
+        seqan3::configuration const cfg =seqan3::search_cfg::max_error_rate{};
```

Comment:
```suggestion
        seqan3::configuration const cfg = seqan3::search_cfg::max_error_rate{};
```
:nail_care: 
Also in a few cases below
Edit: you removed the space almost everywhere. Was this intended? But we do put a space after `=` don't we?